assert/assertion_compare.go

```diff
@@ -378,6 +390,18 @@ func Negative(t TestingT, e interface{}, msgAndArgs ...interface{}) bool {
 	return compareTwoValues(t, e, zero.Interface(), []CompareType{compareLess}, "\"%v\" is not negative", msgAndArgs)
 }
 
+// NonNegative asserts that the specified element is non negative
+//
+//    assert.Negative(t, 0)
+//    assert.Negative(t, 1.23)
+func NonNegative(t TestingT, e interface{}, msgAndArgs ...interface{}) bool {
```

Comment:
I think the functions should be called `Nonnegative()` because this is a single word in the mathematical definition for nonnegative. https://www.merriam-webster.com/dictionary/nonnegative